packages/client/tests/functional/methods/count/tests.ts

```diff
@@ -113,28 +113,28 @@ testMatrix.setupTestSuite(() => {
     } catch (err) {
       expect(err.message).toMatchInlineSnapshot(`
 
-        Invalid \`prisma.user.count()\` invocation:
+                Invalid \`prisma.user.count()\` invocation:
 
-        {
-          select: {
-            _count: {
-              select: {
-        ?       _all?: true,
-        ?       email?: true,
-        ?       age?: true,
-        ?       name?: true,
-                posts: true,
-                ~~~~~
-        ?       id?: true
-              }
-            }
-          }
-        }
+                {
+                  select: {
+                    _count: {
+                      select: {
+                ?       _all?: true,
+                ?       email?: true,
+                ?       age?: true,
+                ?       name?: true,
+                        posts: true,
+                        ~~~~~
+                ?       id?: true
+                      }
+                    }
+                  }
+                }
 
 
-        Unknown field \`posts\` for select statement on model UserCountAggregateOutputType. Available options are listed in green. Did you mean \`id\`?
+                Unknown field \`posts\` for select statement on model UserCountAggregateOutputType. Available options are listed in green. Did you mean \`id\`?
 
-      `)
+            `)
```

Comment:
this indentation change can be reverted I think

Reply:
I agree but I'll merge without changing since this comes from updating the snapshots automatically. It's unfortunate that spacing changes "randomly". The good part is that it doesn't matter for the test in itself, though it's annoying.